Coalition.rte/Devices/Weapons/Shotguns/AutoShotgun.lua

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
Weird whitespace in places
Bad formatting, junk comment marks, bad variable names